lib/hex_web/web/router.ex

```diff
@@ -100,14 +100,15 @@ defmodule HexWeb.Web.Router do
     conn              = fetch_params(conn)
     packages_per_page = @packages
     search            = conn.params["search"] |> safe_query
+    sort              = conn.params["sort"] |> safe_sort
```

Comment:
This is where you should default. Sort on :name by default and on :downloads if we are doing a search.